sunpy/net/vso/vso.py

```diff
@@ -272,60 +273,48 @@ def time_range(self):
                          max(record.time.end for record in self if record.time.end is not None))
 
     def build_table(self):
-        """
-        Create a human readable table.
-
-        Returns
-        -------
-        table : `astropy.table.QTable`
-        """
-        keywords = ['Start Time', 'End Time', 'Source', 'Instrument', 'Type', 'Wavelength']
-        record_items = {}
-        for key in keywords:
-            record_items[key] = []
-
-        def validate_time(time):
-            # Handle if the time is None when coming back from VSO
-            if time is None:
-                return ['None']
-            if record.time.start is not None:
-                return [parse_time(time).strftime(TIME_FORMAT)]
-            else:
-                return ['N/A']
-
-        for record in self:
-            record_items['Start Time'].append(validate_time(record.time.start))
-            record_items['End Time'].append(validate_time(record.time.end))
-            record_items['Source'].append(str(record.source))
-            record_items['Instrument'].append(str(record.instrument))
-            record_items['Type'].append(str(record.extent.type)
-                                        if record.extent.type is not None else ['N/A'])
-            # If we have a start and end Wavelength, make a quantity
-            if hasattr(record, 'wave') and record.wave.wavemin and record.wave.wavemax:
-                unit = record.wave.waveunit
-                # Convert this so astropy units parses it correctly
-                if unit == "kev":
-                    unit = "keV"
-                record_items['Wavelength'].append(u.Quantity([float(record.wave.wavemin),
-                                                              float(record.wave.wavemax)],
-                                                             unit=unit))
-            # If not save None
-            else:
-                record_items['Wavelength'].append(None)
-        # If we have no wavelengths for the whole list, drop the col
-        if all([a is None for a in record_items['Wavelength']]):
-            record_items.pop('Wavelength')
-            keywords.remove('Wavelength')
-        else:
-            # Make whole column a quantity
-            try:
-                with u.set_enabled_equivalencies(u.spectral()):
-                    record_items['Wavelength'] = u.Quantity(record_items['Wavelength'])
-            # If we have mixed units or some Nones just represent as strings
-            except (u.UnitConversionError, TypeError):
-                record_items['Wavelength'] = [str(a) for a in record_items['Wavelength']]
-
-        return Table(record_items)[keywords]
+        # TODO: Workaround for some results having units of kev instead of keV.
+        kev = u.def_unit('kev', u.keV)
+        u.add_enabled_units([kev])
```

Comment:
A hack to get one of the examples working again.

Reply:
ooft, this is a global change of state, we should fix this.